includes/admin/class-wc-admin-log-table-list.php

```diff
@@ -334,6 +334,11 @@ protected function get_items_query_where() {
 			$where_conditions[] = 'source = %s';
 			$where_values[]     = wc_clean( $_REQUEST['source'] );
 		}
+		if ( ! empty( $_REQUEST['s'] ) ) {
+			$where_conditions[] = 'message like %s';
+			$s = wp_unslash( trim( $_REQUEST[ 's' ] ) );
```

Comment:
`wp_unslash()` needs to always comes first.

```
wp> wp_unslash( trim( '\\\\\\\Testing\\\\\\\\' ) )
=> string(11) "\\Testing\\"
wp> wp_unslash( trim( '\\         \\\\\Testing\\\\\       \\' ) )
=> string(11) " \Testing\ "
```

Just copy my example from before that will be just fine.

Reply:
This still should be better:

```php
$where_values[]     = '%' . $wpdb->esc_like( wc_clean( wp_unslash( $_REQUEST['s'] ) ) ) . '%';
```